### ISSUE TITLE
Add more RISC-V projects

### ISSUE THREAD

**Issue by drom** [drom/awesome-riscv#1]

**Comment by mguthaus** [comment]
PicoRV32!

**Comment by drom** [comment]
PR is welcome!

**Comment by zakkak** [comment]
Hello @drom, are software projects that support RISC-V welcome to this repo?

**Comment by drom** [comment]
@zakkak is that an awesome project?

**Comment by zakkak** [comment]
I find it pretty awesome :)
https://github.com/beehive-lab/Maxine-VM

**Comment by drom** [comment]
@zakkak I agree that Maxine-VM is awesome project. It is not too RISC-V specific. But It definitely needs to be in one of these lists:
* https://github.com/aalhour/awesome-compilers
* https://github.com/wdv4758h/awesome-jit

**Comment by zakkak** [comment]
Indeed it's not RISC-V specific (that's also why I initially asked whether software that supports RISC-V is welcome).
It just recently got able to run on RISC-V though! (thanks to @gigiblender)
Thanks for the pointers :) I'll go open some PRs :)

**Comment by drom** [comment]
Maxine-VM running on RISC-V is great news! We need more JITs running on the RISC-V!
Maybe we should start `awesome-riscv-jit` list?

**Comment by zakkak** [comment]
I guess it won't have many candidates for now but why not

**Comment by zignig** [comment]
Minerva.

https://github.com/lambdaconcept/minerva

**Comment by zignig** [comment]
was going to add , been added alredy (https://github.com/drom/awesome-riscv/pull/5)

**Comment by drom** [comment]
Anybody tried, like, can recommend any of these?
* https://github.com/pulp-platform/ariane
* https://github.com/black-parrot/black-parrot
* https://github.com/rsd-devel/rsd
* https://github.com/syntacore/scr1
* https://gitlab.com/sfu-rcl/Taiga
